techbuzz/src/main/java/com/sivalabs/techbuzz/posts/web/controllers/AddVoteController.java

```diff
@@ -28,4 +30,15 @@ public ResponseEntity<Void> createVote(@Valid @RequestBody CreateVoteRequest req
         postService.addVote(createVoteRequest);
         return ResponseEntity.ok().build();
     }
+
+    @PostMapping("/partials/add-vote")
+    @AnyAuthenticatedUser
+    public String createVote(@Valid @RequestBody CreateVoteRequest request, @CurrentUser User loginUser, Model model) {
```

Comment:
You can remove the previous `"/api/votes"` endpoint method and the corresponding test `AddVoteControllerTest` for the new logic.

Reply:
Done